hydrographic_core_msgs/msg/RawBinaryStamped.msg

```diff
@@ -0,0 +1,8 @@
+# message to contain raw binary data from a sensor.
```

Comment:
I know that for some of the basic message building blocks ROS uses the Point/PointStamped convention, but for more complicated messages it seems that `Stamped` tends to just become implicit. Maybe leave it off here?

Reply:
That sounds fine to me.